src/settings-ui/Settings.UI/ViewModels/PluginAdditionalOptionViewModel.cs

```diff
@@ -4,7 +4,9 @@
 
 using System.Collections.Generic;
 using System.ComponentModel;
+using System.Globalization;
```

Comment:
@jaimecbernardo
Do we really need this? Or is this a mistake of VS and can be removed?

Reply:
It's needed for `CultureInfo`.

Reply:
`System.Text.Json.Serialization;` is the one I'm not sure it's needed.

Reply:
I think for the "JSONIgrnore" configuration.

Reply:
Was about to push it as well after testing to make sure :P Thanks for the commit. I'll merge after CI passes (and I look at the PR again)